client/routes/workflow/helpers/index.js

```diff
@@ -25,6 +25,7 @@ export { default as getEventFullDetails } from './get-event-full-details';
 export { default as getEventSummary } from './get-event-summary';
 export { default as getHistoryEvents } from './get-history-events';
 export { default as getHistoryTimelineEvents } from './get-history-timeline-events';
+export { default as getQueryResult } from './get-query-result';
```

Comment:
it is the workflow query view path, right?

Reply:
So this is just the path to the helper in this file. This will allow you to just reference `./helpers` when trying to import `getQueryResult`.
So from `client/routes/workflow/query.vue` you can simply use:
`import { getQueryResult } from './helpers';`